### PR TITLE
Blog post about infinity cosmos project

### PR DESCRIPTION
**Pull request by herostrat**
Hello  @emilyriehl,

I saw your announcement on Zulip. A short disclaimer: I am no mathematician and don't understand anything about this topic. But I am fascinated about Lean and the formalization of stuff which I am way to stupid to ever understand :D

I thought it would be interesting for others to read about your project, a blog post would also be better to link to.
It could be something that could spark additional interest and resources for

I hope you don't mind me creating this draft, please feel free to edit anything as you wish. 
If it is not in your interest to write a blog entry or you have any other problem with the PR please just say so and I will close immediately. I don't want to produce more work or problems, I am just in a way a fan of the things mathlib and math formalization in general does.

Also pinging @dom-verity and @digama0

PS: I plan on proposing blog post for other projects too as surely others other than me also follow mathlib development in a non-contributing fashion and could be kept in the loop about those cool projects...

### CODE REVIEW

**Comment by emilyriehl** [comment]
This is awesome. Thanks for writing it on our behalf. My one suggestion is to add a more introductory first paragraph or two. Eg: 

"Now that Lean's category theory library has reached a certain sophistication, it's natural to ask whether it can be extended to a formalization of (at least certain aspects of) (∞,1)-category theory. (∞,1)-categories --- called "∞-categories" for short --- have objects and morphisms, just like ordinary categories, but composition is only defined up to "homotopy" (encoded by the data of a weakly invertible 2-dimensional morphism) and only associative up to still higher "homotopies" (encoded by the data of weakly invertible n-dimensional morphisms, for all natural numbers n). For such a formalization project to be well-founded, it must start from a rigorous definition of ∞-categories --- for instance using the presentation of ∞-categories as "quasi-categories," a notion which has been defined in Mathlib --- but unfortunately this presentation greatly increases the combinatorial complexity of ∞-categorical definitions and proofs. 

We propose an alternate approach to the formalization of ∞-category theory starting from the axiomatic notion of an ∞-cosmos." 

(then continue with the start of the draft you've produced).

**Comment by emilyriehl** [comment]
@herostrat I tried to push some line edits, but don't seem to have the necessary permissions.  Here's a final version of the text I'd be happy to see go out (a light edit from your initial version).

**Comment by emilyriehl** [comment]
# Introduction

Now that Lean's category theory library has reached a certain sophistication, it's natural to ask whether it can be extended to a formalization of (at least certain aspects of) (∞,1)-category theory. (∞,1)-categories &mdash; often called "[∞-categories](https://www.ams.org/notices/200808/tx080800949p.pdf)" for short &mdash; have objects and morphisms, just like ordinary categories, but composition is only defined up to "homotopy" (encoded by the data of a weakly invertible 2-dimensional morphism) and only associative up to still higher "homotopies" (encoded by the data of weakly invertible n-dimensional morphisms, for all natural numbers n). For such a formalization project to be well-founded, it must start from a rigorous definition of ∞-categories &mdash; for instance using the presentation of ∞-categories as "[quasi-categories](https://ncatlab.org/nlab/show/quasi-category)," a notion which has been defined in Mathlib &mdash; but unfortunately this presentation greatly increases the combinatorial complexity of ∞-categorical definitions and proofs.

We propose an alternate approach to the formalization of ∞-category theory starting from the axiomatic notion of an [∞-cosmos](https://ncatlab.org/nlab/show/infinity-cosmos). An ∞-cosmos is an axiomatization of the universe in which ∞-categories live as objects. The axioms are somewhat ad-hoc and chosen for largely pragmatic reasons. In particular, the axioms are stated in the language of 1-category theory and simplicially enriched category theory &mdash; topics that are well-covered by Mathlib &mdash; rather than in the language of ∞-category theory itself. The theory of ∞-cosmoi enables a "model-independent" development of the theory of ∞-categories. On the one hand, various models such as quasi-categories and complete Segal spaces assemble into ∞-cosmoi, so theorems about ∞-categories in an ∞-cosmos apply to the most popular models. At the same time, the ∞-categorical notions developed in an ∞-cosmos are "synthetic" or "formal" as opposed to the original "analytic" definitions, which were typically expressed in the "coordinates" of a particular model.

Much of the development of the theory of ∞-categories takes place not in the full ∞-cosmos but in a quotient 2-category whose objects are ∞-categories, whose morphisms are ∞-functors, and whose 2-cells are ∞-natural transformations. We call this the homotopy 2-category since it is a quotient of the ∞-cosmos (which is an (∞,2)-category of a particular form) much like an (∞,1)-category has a quotient homotopy (1-)category. For instance, the existing Mathlib notions of equivalences and adjunctions in a bicategory specialize to define equivalences and adjunctions between ∞-categories when interpreted in the homotopy 2-category.

# Initial scope

The initial aims of this project are relatively modest, though with sufficient interest its ambitions could expand considerably. In particular, our initial intention is not to attempt the difficult theorem of proving that the cartesian closed category of quasi-categories defines an example of an ∞-cosmos; among other things, the cartesian closure of this subcategory has not yet been proven. Instead, our intention is to create a large "bounty" to reward anyone who succeeds in this task in the form of a large cache of ∞-categorical theorems that will follow immediately.

# Team behind ∞-Cosmos

This project is being co-lead by [Dominic Verity](https://github.com/dom-verity) and [Mario Carneiro](https://github.com/digama0).

# Contributing

We invite everyone to join us at our [channel at the leanprover Zulip](https://leanprover.zulipchat.com/#narrow/stream/455414-Infinity-Cosmos) for discussions and organisation of development.
If you are interested in contributing to the ∞-cosmos project, we look forward to you joining our conversations.

The project is hosted on [github](https://github.com/emilyriehl/infinity-cosmos), including the [overview](https://emilyriehl.github.io/infinity-cosmos/) and [blueprint](https://emilyriehl.github.io/infinity-cosmos/blueprint/).

**Comment by emilyriehl** [comment]
Could you make this change and then publish or should I go ahead and figure out how to get write/edit access?

**Comment by herostrat** [comment]
Hey Emily, I wasn't expecting this fast of a reply, I am currently traveling and only have my cellphone with me.
I have access to a pc this evening and will push the changes.
Sorry for the delay

**Comment by herostrat** [comment]
Hey Emily, thanks for your patience and the text you provided!

I cannot merge the changes myself, if you are satisfied with the text in general I think the best/fasted way to get it merged is to ask in your zulip channel?

Other than that I wish you and your project all the best, many competent contributors and easy upstreaming  :)

**Comment by emilyriehl** [comment]
I'll do that. Thanks again.

File: posts/infinity-cosmos-announcement.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+# Initial scope
+
+The initial aims of this project are relatively modest, though with sufficient interest its ambitions could expand considerably. In particular, our initial intention is not to attempt the difficult theorem of proving that the cartesian closed category of quasi-categories defines an example of an ∞-cosmos; among other things, the cartesian closure of this subcategory has not yet been proven. Instead, our intention is to create a large "bounty" to reward anyone who succeeds in this task in the form of a large cache of ∞-categorical theorems that will follow immediately.
+
+# Team behind ∞-Cosmos
+
+This project is being co-lead by [Dominic Verity](https://github.com/dom-verity) and [Mario Carneiro](https://github.com/digama0).
```
Comment (kbuzzard) [inline posts/infinity-cosmos-announcement.md:32]:
```suggestion
This project is being co-led by [Dominic Verity](https://github.com/dom-verity) and [Mario Carneiro](https://github.com/digama0).
```

But actually this sentence can be misinterpreted as "there are two leaders of the project, Verity and Carneiro, and Riehl is just somehow doing the announcement of the project" and my understanding is that the truth is "there are three leaders", so I would suggest explicitly mentioning Riehl here too.

**Review by kbuzzard — APPROVED** [review]
This looks fine to me.

File: posts/infinity-cosmos-announcement.md (hunk 1 above)
Comment (herostrat) [reply posts/infinity-cosmos-announcement.md:32]: You are right, thanks.
I also wanted to add Emily, but I wasn't sure how to formulate it in proper english...

My understanding was that Emily is the head of this project and Verity and Carneiro are co-leading it.
What about:

This project was started/envisioned/initiated by Emily and is beeing co-led by Verity and Carneiro.

Sorry if it sounds rough, english is not my first language

**Pull request merged**